docs/src/content/docs/commands/JSON.ARRPOP.md

```diff
@@ -0,0 +1,80 @@
+---
+title: JSON.ARRPOP
+description: Documentation for the DiceDB command JSON.ARRPOP
+---
+
+The `JSON.ARRPOP` command in DiceDB is used to pop an element from JSON array located at a specified path within a JSON document. This command is part of the DiceDBJSON module, which provides native JSON capabilities in DiceDB.
+
+## Syntax
+
+```plaintext
+JSON.ARRPOP key [path [index]]
+```
+
+## Parameters
+| Parameter | Description                                                                   | Type    | Required |
+|-----------|-------------------------------------------------------------------------------|---------|----------|
+| `key`     | The key under which the JSON document is stored.                              | String  | Yes      |
+| `path`    | The JSONPath expression that specifies the location within the JSON document. | String  | Yes      |
+| `index`   | The index of the element that needs to be popped from the JSON Array at path. | Integer | No       |
+
+## Return Value
+
+- `string, number, object, array, boolean`: The element that is popped from the JSON Array.
+- `Array`: The elements that are popped from the respective JSON Arrays.
+
+## Behaviour
+
+When the `JSON.ARRPOP` command is executed, the specified element is popped from the array located at the given index at the given path within the JSON document stored under the specified key. If the path does not exist or does not point to an array, an error will be raised.
+
+## Error Handling
+
+- `(error) ERR key does not exist`: Raised if the specified key does not exist in the DiceDB database.
+- `(error) ERR path %s does not exist`: Raised if the specified path, s, does not exist within the JSON document.
+- `(error) ERR path is not an array`: Raised if the specified path does not point to a JSON array.
+- `(error) WRONGTYPE Operation against a key holding the wrong kind of value`: Raised if any of the provided JSON values are not valid JSON.
+
```

Comment:
Please follow below bullet structure for errors in both files
1. `Wrong type of value or key`:

   - Error Message: `(error) WRONGTYPE Operation against a key holding the wrong kind of value`
   - Occurs when attempting to use the command on a key that contains a non-string value.